pkg/cache/secrets.go

```diff
@@ -1,10 +1,11 @@
 package cache
 
 import (
-	"k8s.io/klog/v2"
 	"sync"
 
-	sharev1alpha1 "github.com/openshift/csi-driver-projected-resource/pkg/api/projectedresource/v1alpha1"
+	"k8s.io/klog/v2"
+
+	sharev1alpha1 "github.com/openshift/csi-driver-shared-resource/pkg/api/projectedresource/v1alpha1"
```

Comment:
```suggestion
	sharev1alpha1 "github.com/openshift/csi-driver-shared-resource/pkg/api/sharedresource/v1alpha1"
```

Reply:
ditto - not changing the api groups/names in this PR.